cross/jellyfin/Makefile

```diff
@@ -10,12 +10,12 @@ HOMEPAGE = https://jellyfin.org
 COMMENT  = The Free Software Media System. It is an alternative to the proprietary Emby and Plex.
 LICENSE  = GPLv2
 
-DOTNET_VERSION = 6.0
+DOTNET_VERSION = 8.0
 BUILD_DEPENDS = native/dotnet-sdk-$(DOTNET_VERSION)
 
 DOTNET_OUTPUT_PATH = share
 DOTNET_PACKAGE_NAME = Jellyfin.Server
 # avoid the creation of pdb files
 DOTNET_BUILD_PROPERTIES = -p:DebugSymbols=false -p:DebugType=none
 
-include ../../mk/spksrc.cross-dotnet.mk
+include ../../mk/spksrc.cross-dotnet.mk
```

Comment:
please revert this change (keep the final line break)

Reply:
Already solved